common/persistence/class.SqlKvDriver.php

```diff
@@ -181,25 +182,33 @@ public function del($id)
     /**
      * Increment existing value
      * @param string $id
-     * @return mixed
+     * @return boolean
+     *
+     * @throws common_Exception
      */
     public function incr($id)
     {
-        $params = [':id' => $id];
-        $statement = 'UPDATE kv_store SET kv_value = kv_value + 1 WHERE kv_id = :id';
-        return $this->sqlPersistence->exec($statement, $params);
+        $value = filter_var($this->get($id), FILTER_VALIDATE_INT);
```

Comment:
Isn't that risky without transactions?

Reply:
as discussed with @zagovorichev , Im going to use update query instead of get/set